lms/static/scripts/frontend_apps/components/test/LMSGrader-test.js

```diff
@@ -81,6 +97,97 @@ describe('LMSGrader', () => {
     );
   });
 
+  it('orders the students by last name (if it exists), then first name', () => {
+    // Un-order students
+    fakeStudents = [
+      {
+        displayName: 'Student Beta',
+      },
+      {
+        displayName: 'Students Beta',
+      },
+      {
+        displayName: 'Beta',
+      },
+      {
+        displayName: 'student Beta',
+      },
+      {
+        displayName: 'Student Delta',
+      },
+      {
+        displayName: 'Student Alpha',
+      },
+      {
+        displayName: 'Alpha',
+      },
+    ];
+    const wrapper = renderGrader();
+    const orderedStudentNames = getDisplayNames(wrapper);
+    assert.match(
+      [
+        'Alpha',
+        'Beta',
+        'Student Alpha',
+        'Student Beta',
+        // 'student Beta' ties with 'Student Beta',
+        // but since 'Student Beta' came first, it wins.
+        'student Beta',
+        'Student Delta',
+        'Students Beta',
+      ],
+      orderedStudentNames
+    );
+  });
+
+  it('updates the order if the `students` change', () => {
+    // Un-order students
+    fakeStudents = [
+      {
+        displayName: 'Beta',
+      },
+      {
+        displayName: 'Alpha',
+      },
+    ];
+    const wrapper = renderGrader();
+    let orderedStudentNames = getDisplayNames(wrapper);
+    assert.match(['Alpha', 'Beta'], orderedStudentNames);
+    // New list of students
+    wrapper.setProps({
+      students: [
+        {
+          displayName: 'Beta',
+        },
+        {
+          displayName: 'Gamma',
+        },
+      ],
+    });
+    orderedStudentNames = getDisplayNames(wrapper);
+    assert.match(['Beta', 'Gamma'], orderedStudentNames);
+  });
+
+  it('sorts an undefined displayName first', () => {
```

Comment:
Perhaps this could be worded as "puts students with empty `displayName`s at the beginning of sorted students"